File: 181 excersises/178.cpp
#include <iostream>
using namespace std;

int sumarElementos(int* Lista, int tamaño) {
    int suma = 0;
    int* ptr = Lista;
    
    for(int i = 0; i < tamaño; i++) {
        suma += *ptr; 
        ptr++;      
    }
    
    return suma;
}

int main() {
    int tamaño;
    cout << "Ingresa el tamaño de la lista: ";
    cin >> tamaño;
    
    int* lista = new int[tamaño];
    
    cout << "Ingresa los " << tamaño << " valores:\n";
    for(int i = 0; i < tamaño; i++) {
        cout << "valores " << i+1 << ": ";
        cin >> lista[i];
    }
    
    int total = sumarElementos(lista, tamaño);
    cout << "La suma de los elementos es: " << total << endl;

    delete[] lista;
    
    return 0;
}

File: 181 excersises/178.py
def sumar_elementos(Lista):
    suma = 0
    indice = 0
    
    while indice < len(Lista):
        suma += Lista[indice]
        indice += 1
    
    return suma

tamaño = int(input("Ingresa el tamaño de la Lista: "))
Lista = []

print(f"Ingresa los {tamaño} valores:")
for i in range(tamaño):
    elemento = int(input(f"valores {i+1}: "))
    Lista.append(elemento)

total = sumar_elementos(Lista)
print(f"La suma de los valores es: {total}")